amazon-s3/6.0/modules/ROOT/pages/amazon-s3-connector-config-topics.adoc

```diff
@@ -0,0 +1,183 @@
+= Amazon S3 Connector 6.0 Additional Configuration
+:page-aliases: connectors::amazon/amazon-s3-connector-config-topics.adoc
+
+== Create Object Operation
+
+In the `Create Object` operation, set the `Content Length` to a value greater than zero. If the `Content Length` is set to zero (0), the `Create Object` operation creates a zero-byte object.
+
+== Use an AWS KMS Master Key
```

Comment:
Now there is not a single field for this, but multiple fields inside the ObjectMetadata object.